### PR TITLE
Change the interface and finish the whole function.

### PR DESCRIPTION
**Pull request by MCTBL**
Refine #419 pr

### CODE REVIEW

File: dependencies.gradle [hunk 1]
```diff
@@ -3,6 +3,11 @@
 dependencies {
     implementation('com.github.GTNewHorizons:NotEnoughItems:2.4.12-GTNH:dev')
     implementation('curse.maven:cofh-core-69162:2388751')
+    
+    runtimeOnly('com.github.GTNewHorizons:Baubles:1.0.1.16:dev')
```
Comment (Laiff) [inline dependencies.gradle:7]: It is better to use `runtimeOnlyNonPublishable` here

File: src/main/java/appeng/me/cache/GridStorageCache.java [hunk 2]
```diff
@@ -398,12 +396,41 @@ private void updateBytesInfo() {
...
                     for (IMEInventoryHandler<?> meih : icp.getCellArray(StorageChannel.FLUIDS)) {
+                        // exclude void cell
+                        if (((DriveWatcher<IAEItemStack>) meih).getInternal() instanceof ICellCacheRegistry iccr) {
+                            // exclude creative cell
+                            if (iccr.canGetInv()) {
+                                if (iccr.getCellType() == ICellCacheRegistry.TYPE.FLUID) {
+                                    fluidBytesTotal += iccr.getTotalBytes();
+                                    fluidBytesUsed += iccr.getUsedBytes();
+                                    switch (iccr.getCellStatus()) {
+                                        case 1 -> fluidCellG++;
```
Comment (Laiff) [inline src/main/java/appeng/me/cache/GridStorageCache.java:413]: Lets introduce readable constants for this statuses to get rid of magic numbers

**Review by Laiff — CHANGES_REQUESTED** [review]
Small improvements only, in other, and can you rebuild git history to get rid of unnecessary commits thats already merged

File: dependencies.gradle (hunk 1 above)
Comment (MCTBL) [reply dependencies.gradle:7]: All these four switches to `runtimeOnlyNonPublishable`?

**Review by Laiff — APPROVED** [review]

**Pull request merged**